packages/desktop/src/renderer/app/components/route-response-rules/route-response-rules.component.html

```diff
@@ -112,6 +112,16 @@
               "
             />
 
+            <button
+              type="button"
+              ngbTooltip="toggle enable"
+              class="btn btn-link btn-icon"
+            >
+              <app-svg
+                icon="check"
+              />
+            </button>
+
```

Comment:
One remark here (but it's a WIP and you may have already figured that), you will have to use a proper checkbox and link it to the reactive form like the other inputs. [Example here](https://github.com/mockoon/mockoon/blob/ea2695bd49d90d0dd2db648f4b1e8153d825d935/packages/desktop/src/renderer/app/components/environment-proxy/environment-proxy.component.html#L8-L16).

Reply:
Hi @255kb thanks for the help. Sorry if I haven't been working on it lately, something came up and I got busy. I'll try to get some activity going on this weekend.

Also, I didn't know Github notifies the maintainers of any created PR (even in Forked)

Reply:
No problem, I just wanted to give some early feedback to avoid you losing time. 
When you open a PR it's actually in the repo you forked, not in the fork. That's why I'm seeing it :)